src/App.js

```diff
@@ -12,21 +12,24 @@ import {
 } from './Pages';
 import './App.css';
 import Footer from './Components/Footer/Footer';
-import Header from './Components/Navigation/Header';
 
 function App() {
+  // new stuff
+  // const [image, setImage] = useState(""); 
+  // const addImageHandler = (image) => {
+  //   setImage((prevImage)=> {
+  //     return image; 
+  //   })
```

Comment:
Could you clarify if this sprint is separate from the map itself? If not can we add  a route for the milestone map?